.github/ISSUE_TEMPLATE/release-tracker.yml

```diff
@@ -10,6 +10,12 @@ body:
       label: AMP Version
       description: Replace VERSION with AMP version.
       value: |
+        <!-- ----------------------------------------------------------------------- -->
+        <!-- ----------------------------- DO NOT EDIT ----------------------------- -->
+        <!-- ----------------------------------------------------------------------- -->
+        <!-- This issue is managed automatically by the release-tagger.yml workflow. -->
+        <!-- Manually editing it can break the automation! ------------------------- -->
+        <!-- ----------------------------------------------------------------------- -->
```

Comment:
Drive by comment: This seems to contradict the description above, which asks the on-duty to replace `VERSION` with the actual version. Maybe the rest of the instructions in this template also need to be updated to reflect the fact that tracker issues are auto-created and auto-updated?